Resources/public/css/styles.css

```diff
@@ -216,6 +216,11 @@ body.fixed .content-header .navbar {
     padding: 15px;
 }
 
+.sidebar-menu li.keep-open > .treeview-menu {
+    display: block !important;
+    height: auto !important;
```

Comment:
It is require to set height? I am trying and I only need to set display: block to show the inner menu